plutus-ledger-api/testlib/PlutusLedgerApi/Test/EvaluationContext.hs

```diff
@@ -1,29 +1,119 @@
 -- editorconfig-checker-disable-file
+{-# LANGUAGE TypeFamilies  #-}
+{-# LANGUAGE TypeOperators #-}
 module PlutusLedgerApi.Test.EvaluationContext
-    ( costModelParamsForTesting
-    , evalCtxForTesting
+    ( v1_costModelParamsForTesting
+    , v2_costModelParamsForTesting
+    , v3_costModelParamsForTesting
     ) where
 
-import PlutusCore.Evaluation.Machine.CostModelInterface as Plutus
-import PlutusCore.Evaluation.Machine.ExBudgetingDefaults qualified as Plutus
-import PlutusLedgerApi.Common
+import PlutusCore.Evaluation.Machine.BuiltinCostModel
+import PlutusCore.Evaluation.Machine.CostModelInterface
+import PlutusCore.Evaluation.Machine.ExBudgetingDefaults
+import PlutusCore.Evaluation.Machine.MachineParameters
 import PlutusPrelude
+import UntypedPlutusCore.Evaluation.Machine.Cek.CekMachineCosts
 
-import Data.Either.Extras
+import PlutusLedgerApi.Common as Common
+import PlutusLedgerApi.V1 qualified as V1
+import PlutusLedgerApi.V2 qualified as V2
+import PlutusLedgerApi.V3 qualified as V3
+
+import Barbies
+import Data.Aeson
+import Data.Functor.Identity
+import Data.Map qualified as Map
 import Data.Maybe
 
-{-| only for testing purposes: The default, raw cost model params.
+-- | Example values of costs for plutus-v3 , in expected ledger order, i.e. can be used in testing
```

Comment:
```suggestion
-- | Example values of costs for @PlutusV3@ , in expected ledger order. Suitable to be used in testing.
```
etc

Reply:
Would they be unsuitable if they were in the wrong order?

Reply:
>  It is not super bad, but it is not "representable" of the costs.

Right, so they _could_ just be random numbers if you only want to check that this code does what you expect, but using the correct numbers doesn't do any harm.  I just wondered if it was doing something that I hadn't spotted that needed the actual values.